### PR TITLE
Fix(plugins): Fix cv_zscaler_endpoints lookup plugin

### PR DESCRIPTION
**Pull request by ClausHolbechArista**
## Change Summary

Fix cv_zscaler_endpoints lookup plugin

## Component(s) name

Lookup plugin `arista.avd.cv_zscaler_endpoints`

## Proposed changes

- Fix various API changes
- Fix various bugs

## How to test

Added a new integration test which will only work when manually supplying a token.

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/plugins/lookup/cv_zscaler_endpoints.py [hunk 1]
```diff
@@ -21,22 +21,32 @@
     type: str
     description: CV server.
     required: true
+    vars:
+      - name: cv_server
```
Comment (gmuloc) [inline ansible_collections/arista/avd/plugins/lookup/cv_zscaler_endpoints.py:25]: is this required? I don't see this field in ansible doc: https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-fields and it looks redundant and does not seem to impact our doc.

File: ansible_collections/arista/avd/tests/integration/targets/cv_zscaler_endpoints/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+- name: Test cv_zscaler_endpoints lookup plugin
+  tags: [always]
+  delegate_to: localhost
+  when: cv_token is arista.avd.defined
+  vars:
+    # This is not working with ansuble-test, so adjust for manual testing.
```
Comment (gmuloc) [inline ansible_collections/arista/avd/tests/integration/targets/cv_zscaler_endpoints/tasks/main.yml]:
```suggestion
    # This is not working with ansible-test, so adjust for manual testing.
```

File: ansible_collections/arista/avd/plugins/lookup/cv_zscaler_endpoints.py (hunk 1 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/plugins/lookup/cv_zscaler_endpoints.py:25]: This is required for the arguments to be picked up from vars (so you can call the lookup plugin without args). I added it for a reason ;)

Comment (gmuloc) [reply ansible_collections/arista/avd/plugins/lookup/cv_zscaler_endpoints.py:25]: ok now that I understand what it means!

**Review by gmuloc — APPROVED** [review]
LGTM

**Pull request merged**